src/SConscript

```diff
@@ -6,42 +6,42 @@ from glob import glob
 
 Import("*")
 
+#
+# load list of sources from the file in scons/sources/
```

Comment:
Whoops, that where it was when I first submitted this and it was just for scons.  WIll fix.